tests/pipelines/kit/etcd-list-job-pipeline.yaml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: etcd-list-job-pipeline
+  namespace: tekton-pipelines
+spec:
+  params:
+  - name: name
+    description: The name of the test cluster.
+    default: "guest"
+  - name: scheduler-replicas
```

Comment:
Lets syncup offline about all these params, most of this should be default.

Reply:
Agree. Please see the new version.